Projects/Modules/Documentarian.MicrosoftDocs/Source/Public/Functions/Get-CmdletXref.ps1

```diff
@@ -0,0 +1,50 @@
+﻿function Get-CmdletXref {
+    <#
+    .SYNOPSIS
+    Gets a cross-reference link for a command.
+    #>
+    [CmdletBinding()]
+    [OutputType('System.String[]')]
+    param(
+        # The name of the command to get a cross-reference link for.
+        [Parameter(Mandatory, ValueFromPipeline)]
+        [string[]]$Name
+    )
```

Comment:
Nit, but probably useful - Can this be updated to support piping a set of command info objects from `Get-Command`, like:

```powershell
Get-Command Convert*-Json | Get-CmdletXref | Set-Clipboard
```

That might complicate things a bit into two different parameter sets.

Reply:
Good idea. File an issue ;)